interceptors/logging/interceptors.go

```diff
@@ -26,12 +27,15 @@ type reporter struct {
 	typ             interceptors.GRPCType
 	service, method string
 
+	requestLoggerDecider bool
```

Comment:
I'd rather name this to 
```suggestion
	requestLoggerDecision bool
```